### PR TITLE
Use pure JavaScript instead of jQuery [6]

### PR DESCRIPTION
**Pull request by 1v9**

### CODE REVIEW

File: source/css/_common/components/post/post-expand.styl [hunk 1]
```diff
@@ -49,7 +49,7 @@
     margin-bottom: 20px;
   }
 
-  .video-container {
```
Comment (stevenjoezhang) [inline source/css/_common/components/post/post-expand.styl]: It should be `.video-container`, see
https://github.com/hexojs/hexo/blob/0b26940f7e0a35f6937b944012468bbf0d844010/lib/plugins/tag/vimeo.js
https://github.com/hexojs/hexo/blob/0b26940f7e0a35f6937b944012468bbf0d844010/lib/plugins/tag/youtube.js

File: source/js/utils.js [hunk 2]
```diff
@@ -68,16 +74,20 @@ NexT.utils = {
    * One-click copy code support.
    */
   registerCopyCode: function() {
-    $('.highlight').not('.gist .highlight').each((i, e) => {
-      function initButton(button) {
+    document.querySelectorAll('.highlight:not(.gist)').forEach(e => {
```
Comment (stevenjoezhang) [inline source/js/utils.js]: `.highlight:not(.gist)` or `:not(.gist) .highlight`?

File: source/js/utils.js [hunk 3]
```diff
@@ -68,16 +74,20 @@ NexT.utils = {
...
+      const initButton = button => {
         if (CONFIG.copycode.style === 'mac') {
-          button.html('<i class="fa fa-clipboard"></i>');
+          button.innerHTML = '<i class="fa fa-clipboard"></i>';
         } else {
-          button.text(CONFIG.translation.copy_button);
+          button.textContent = CONFIG.translation.copy_button;
```
Comment (stevenjoezhang) [inline source/js/utils.js]: `innerText` or `textContent`?

File: source/js/utils.js [hunk 4]
```diff
@@ -68,16 +74,21 @@ NexT.utils = {
    * One-click copy code support.
    */
   registerCopyCode: function() {
-    $('.highlight').not('.gist .highlight').each((i, e) => {
-      function initButton(button) {
+    document.querySelectorAll('.highlight').forEach(e => {
+      if (e.classList.contains('tab-size')) return;
```
Comment (1v9) [inline source/js/utils.js]: bad and need a native selector

**Comment by 1v9** [comment]
gist -------> `table.highlight.xyz`

**Pull request merged**